dbt/__init__.py

```diff
@@ -0,0 +1,3 @@
+from pkgutil import extend_path
```

Comment:
I am not sure to understand why we should include this code ? But I have never used VSCode before 🤔

Reply:
I'm not sure neither that is is necessary. in Pycharm/IntellJ is possible to setup the a specific folder as `src`, maybe there is something similar in VScode

Reply:
I'm not an expert of python packages but [this](https://docs.python.org/3/library/pkgutil.html#pkgutil.extend_path) is the only solution I found to fix pylance errors in vscode without introducing errors in Pycharm. In Pycharm works well out of the box.

If we can fix with a vscode settings I'm also fine but I haven't found a solution yet.

Reply:
Re implications: adding an __init__.py will change this to a different type of namespace package: https://packaging.python.org/en/latest/guides/packaging-namespace-packages/

Given that dbt also uses this kind of "pkgutils" namespace type: https://github.com/dbt-labs/dbt-core/blob/main/core/dbt/__init__.py

Adding this seems to be the right way:

> Every distribution that uses the namespace package must include an identical __init__.py. If any distribution does not, it will cause the namespace logic to fail and the other sub-packages will not be importable. Any additional code in __init__.py will be inaccessible.

Reply:
Thanks, @jankatins. I'll keep it then ;)